doc/lsst.pipe.base/creating-a-pipeline.rst

```diff
@@ -451,15 +451,57 @@ class name of the python camera object.  For instance, for an ``obs_subaru``
 
   instrument: lsst.obs.subaru.HyperSuprimeCam
```

Comment:
I don't see any documentation of what order config settings are applied in (field defaults, `setDefaults`, general obs overrides, instrument obs overrides, top-level pipelines, included pipelines, and maybe other stuff I'm forgetting). It would be hard for the reader to make informed decisions without that kind of big picture.